app/views/evaluations/_form.html.erb

```diff
@@ -1,77 +1,83 @@
-<%= form_with(model: @evaluation, data: { controller: "evaluation modal" }) do |form| %>
-  <% if @evaluation.errors.any? %>
-    <div style="color: darkred">
-      <div class="usa-alert usa-alert--error" role="alert">
-        <div class="usa-alert__body">
-          <h2 class="usa-alert__heading"><%= pluralize(@evaluation.errors.count, "error") %> prohibited this evaluation from being saved:</h2>
-          <ul>
-            <% @evaluation.errors.each do |error| %>
-              <li><%= error.full_message %></li>
-            <% end %>
-          </ul>
+<div data-controller="evaluator-recusal-modal">
+  <%= render 'evaluator_recusal_button', assignment: local_assigns[:assignment] if local_assigns[:assignment] %>
+
+  <%= form_with(model: @evaluation || Evaluation.new, data: { controller: "evaluation modal" }) do |form| %>
```

Comment:
don't think the fallback is needed since @evaluation is defined in all the routes that render the form
```suggestion
  <%= form_with(model: @evaluation, data: { controller: "evaluation modal" }) do |form| %>
```